src/main/java/seedu/recipe/logic/commands/AddFormCommand.java

```diff
@@ -55,7 +55,7 @@ public CommandResult execute(Model model) throws CommandException {
             model.addRecipe(recipeToAdd);
             return new CommandResult(String.format(MESSAGE_SUCCESS, recipeToAdd.getName()));
         } catch (ParseException e) {
-            throw new CommandException(MESSAGE_PARSE_RECIPE);
+            throw new CommandException(e.getMessage());
```

Comment:
Tested with every field because I was worried the message might be too technical for normal users, but the messages are well-done. 
One problem: Invalid input in the `portion` field sometimes returns `unknown command` or has no error message instead of command format error. E.g. inputting `1swag` in `portion `can give `unknown command` or no error message while not adding.

Reply:
Same issue occurs in the `name`, `duration` and `portion` fields for `edit form`, except 
deleting the field returns the format error message. E.g. deleting `10.0 minutes` under `duration` returns `A Recipe's duration should consist of an integer/fraction/decimal section and an alphabetic time unit (spaces allowed)`. And `deleting name` should return a more informative message, like 'recipe name cannot be deleted, to delete the recipe, press d on the recipe card.`
No issues on the steps, ingredients and tags though.

Reply:
On another note, ps for spamming but "1 serving" does not work also